### ISSUE TITLE
Feature suggestion: setup voting

### ISSUE THREAD

**Issue by maliMirkec** [wesbos/awesome-uses#384]
Maybe allow only liking the setup, without disliking.

Originally proposed by @palashmon in this pull request: #372

**Comment by palashmon** [comment]
Agreed. We can only have a liking feature for now and add a new section somewhere showing top 5 liked pages overall. In future, we can add more features like top 5 by tags, country, etc. We can also do top 5 in current month, year, all-time, etc.

**Comment by iamandrewluca** [comment]
I think this feature will imply authentication.

**Comment by sam-cross** [comment]
Maybe not authentication but it would require some sort of server-side tracking. The site is hosted as a static page, so unfortunately I don't see this being possible.

**Comment by HugoDF** [comment]
@sam-cross We can do this using Netlify Functions, would need some sort of database of course & authentication.

**Comment by sam-cross** [comment]
@HugoDF [This](https://www.netlify.com/blog/2018/07/09/building-serverless-crud-apps-with-netlify-functions-faunadb/) may be a pretty good option, in that case

**Comment by HugoDF** [comment]
To stick with the mainly static mentality:
- GitHub oauth
- FaunaDB or Netlify Forms for persistence
- Couple of backend functions to stitch it together
